app/views/Settings/ENDebugMenu.tsx

```diff
@@ -105,11 +105,7 @@ export const ENDebugMenu = ({ navigation }: ENDebugMenuProps): JSX.Element => {
       onBackPress={backToSettings}>
       <ScrollView>
         <Section>
-          <Item
-            label='Toggle Exposure State'
-            onPress={handleOnPressToggleExposure}
-            last
-          />
+          <Item label='Reset Exposures' onPress={handleOnPressResetExposures} />
```

Comment:
I know you have a worthy vendetta against this prop, but for styling purposes: `last` :)